src/dmd/mars.d

```diff
@@ -24,7 +24,7 @@ import core.stdc.string;
 
 import dmd.arraytypes;
 import dmd.astcodegen;
-import dmd.gluelayer;
+import dmd.glue : generateCodeAndWrite;
```

Comment:
Judging by the failures. Put this in version(!NoMain)

Reply:
Thanks, Will try